buildSrc/build.gradle.kts

```diff
@@ -4,9 +4,8 @@ plugins {
 
 repositories {
     mavenLocal()
-    jcenter()
-    maven(url = "https://dl.bintray.com/kotlin/kotlin-eap")
     mavenCentral()
+    jcenter() // shadow plugin
```

Comment:
use `gradlePluginPortal()` instead. (johnrengelman/shadow#641)

Reply:
Thank you.